### PR TITLE
docs(README): add quick start section with a simple code example

### PR DESCRIPTION
**Pull request by ayoubmehd**
# add quick start section 

## Problem
Add a quick start section #1588 

## Solution

README

Add a quick start section with a link at the end to the Example section.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -59,6 +60,21 @@ For information about terms used in this document see: [terminology](./docs/term
 npm install commander
 ```
 
+## Quick Start
+
+```js
+const { program } = require("commander");
+
+program
+    .command('list') // The name of your command
```
Comment (shadowspawn) [inline Readme.md:69]: I see potential for confusion in the comment for new user between giving the program a name, and adding a new subcommand?

Perhaps: add a subcommand named "list"

**Review by shadowspawn — CHANGES_REQUESTED** [review]
Cosmetic: be consistent with whether starting comments with lowercase or uppercase letter

The big two elements are commands and options, and programs with only options are a real use case. I would like an example of an option too (on `list` command).

File: Readme.md (hunk 1 above)
Comment (ayoubmehd) [reply Readme.md:69]: Yeah I see

**Comment by shadowspawn** [comment]
Do you think having both a small example at the top and the longer example at the bottom is useful? Or should the example be pulled up to the top of the file?

**Comment by ayoubmehd** [comment]
@shadowspawn The Quick Start examples are for new users, the bottom examples will be just fine there, those are more 'advanced'

**Comment by shadowspawn** [comment]
I was wondering about showing some example usage, but think focus on simple code could be good.

(Still reading and thinking abut Quick Starts.)

**Comment by shadowspawn** [comment]
This has been a useful exercise for me to see a candidate Quick Start, but it isn't adding enough value in current form. I'll close the PR and keen the issue open, and see if any other issues come up that would be helped by a Quick Start.

Thank you  for your contributions.

**Pull request closed without merging**